core/src/main/java/io/questdb/griffin/engine/functions/bool/BetweenTimestampFunctionFactory.java

```diff
@@ -90,9 +91,14 @@ public boolean getBool(Record rec) {
 
             return from <= timestamp && timestamp <= to;
         }
+
+        @Override
+        public void toPlan(PlanSink sink) {
+            sink.val(left).val(" between ").val(from).val(" and ").val(to);
+        }
     }
 
-    private static class VarBetweenFunction extends NegatableBooleanFunction implements TernaryFunction {
+    private static class VarBetweenFunction extends BooleanFunction implements TernaryFunction {
```

Comment:
why is this function non-negatable?

Reply:
NegatableFunction is meant for regular >, >=, = operators . 
Other functions (including between) require using NOT operator/ NotFunctionFactory.Func function.